src/Microsoft.AspNet.Mvc.Razor/RazorPage.cs

```diff
@@ -209,30 +209,15 @@ private IViewBufferScope BufferScope
         /// </remarks>
         public void StartTagHelperWritingScope()
         {
-            var buffer = new ViewBuffer(BufferScope, Path);
-            StartTagHelperWritingScope(new HtmlContentWrapperTextWriter(buffer, Output.Encoding));
-        }
-
-        /// <summary>
-        /// Starts a new writing scope with the given <paramref name="writer"/>.
-        /// </summary>
-        /// <remarks>
-        /// All writes to the <see cref="Output"/> or <see cref="ViewContext.Writer"/> after calling this method will
-        /// be buffered until <see cref="EndTagHelperWritingScope"/> is called.
-        /// </remarks>
-        public void StartTagHelperWritingScope(TextWriter writer)
```

Comment:
@NTaylorMullen it doesn't look like we use this method (none of the functional tests rely on this). Is there a scenario I'm missing that isn't covered by the tests?